addons/im_livechat/report/im_livechat_report_channel_views.xml

```diff
@@ -54,18 +79,30 @@
             </field>
         </record>
 
+        <record id="im_livechat_report_channel_view_graph" model="ir.ui.view">
```

Comment:
I'm not sure what's happening with the diff, but you could try to keep the record in the same place so it doesn't show as a new one.